apps/staking/app/stake/[address]/ManageStake.tsx

```diff
@@ -29,7 +29,7 @@ export function ManageStake({ contract }: { contract: ContributorContractInfo })
   const isOperator = areHexesEqual(contract.operator_address, address);
   const isFinalized = contract.status === CONTRIBUTION_CONTRACT_STATUS.Finalized;
 
-  const { totalStaked } = getContributionRangeFromContributors(contract.contributors);
+  const totalStaked = getTotalStaked(contract.contributors);
```

Comment:
Unused?

Reply:
ah yes oops, its generated in the info comp now, removed